account_move_line_stock_info/models/account_move_line.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = 'account.move.line'
+
+    stock_move_id = fields.Many2one(comodel_name='stock.move',
+                                    string='Stock Move', copy=False)
```

Comment:
@lreficent Since you have declared an inverse on this field into stock.move you must add `index=True` to avoir performance issue